Calibration/HcalCalibAlgos/plugins/AnalyzerMinbias.cc

```diff
@@ -245,77 +239,71 @@ void AnalyzerMinbias::endJob() {
 // ------------ method called to produce the data  ------------
 
 void AnalyzerMinbias::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup) {
-  rnnum = (float)iEvent.run();
+  rnnum_ = (float)iEvent.run();
   const HcalRespCorrs* myRecalib = nullptr;
   if (theRecalib_) {
     myRecalib = &iSetup.getData(tok_respCorr_);
   }  // theRecalib
 
-  edm::Handle<HBHERecHitCollection> hbheNormal;
-  iEvent.getByToken(tok_hbheNormal_, hbheNormal);
+  const edm::Handle<HBHERecHitCollection> hbheNormal = iEvent.getHandle(tok_hbheNormal_);
   if (!hbheNormal.isValid()) {
     edm::LogVerbatim("AnalyzerMB") << " hbheNormal failed";
```

Comment:
```suggestion
    edm::LogVerbatim("AnalyzerMinbias") << " hbheNormal failed";
```

This is the correct name of the package